profiles.yml

```diff
@@ -0,0 +1,13 @@
+mozdbt:
+  outputs:
+    dev:
+      dataset: tmp
```

Comment:
Currently this is the only configured profile so everything is deployed to `tmp` at the moment.